### PR TITLE
Add Italian SSCD interface

### PR DESCRIPTION
**Pull request by forsthug**
- [x] You've read the [Contributor Guide](https://github.com/fiskaltrust/.github/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/fiskaltrust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

{Summary of the changes}

## Description

{Detail}

Fixes #{issue number}

### CODE REVIEW

File: src/fiskaltrust.ifPOS/v1/it/Models/RecMessage.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+﻿using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.it
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.ifPOS/v1/it/Models/RecMessage.cs]: Again very epson-specific, but let's keep this for now and see how the other interfaces look, I guess.

File: src/fiskaltrust.ifPOS/v1/it/Models/FiscalReceiptRequest.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        [DataMember]
+        public List<RecSubtotalAdjustment> RecSubtotalAdjustments { get; set; }
+
+        /// <summary>
+        /// printRecTotal: One or more commands can be sent; more than one means that the payment is composed of several 
+        /// partial payments.In this case, once the total has been reached or exceeded, no more payment
+        /// commands can be sent
+        /// </summary>
+        [DataMember]
+        public List<RecTotal> RecTotals { get; set; }
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.ifPOS/v1/it/Models/FiscalReceiptRequest.cs:55]: I'd maybe omit this completely for now, what do you think @forsthug ?

File: src/fiskaltrust.ifPOS/v1/it/Models/States.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
+﻿using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.it
+{
+    /// <summary>
+    ///  rtMainStatus indicates the RT main status
+    /// </summary>
+    [DataContract]
+    public enum MainState
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.ifPOS/v1/it/Models/States.cs:9]: The existence of these states make me think that there probably are some "lifecycle" methods, e.g. to initialize/decommission the printer. Let's fine out if these can be set programmatically 😁

File: src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    /// <summary>
+    /// The interface to communicate with fiskaltrust's SCUs for Italy.
+    /// </summary>
+    [ServiceContract]
+    public interface IITSSCD
+    {
+        /// <summary>
+        /// Returns printer information 
+        /// </summary>
+        [OperationContract(Name = "v2/GetPrinterInfo")]
```
[the remark this answers is not in the document]
Comment (forsthug) [reply src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs:15]: In ME is v2, that why i took it,

File: src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        /// Send FiscalReceipt to the printer.
+        /// </summary>
+        [OperationContract(Name = "v2/FiscalReceipt")]
+        Task<FiscalReceiptResponse> FiscalReceiptAsync(FiscalReceiptRequest request);
+
+        /// <summary>
+        /// StartExportSession
+        /// </summary>
+        [OperationContract(Name = "v2/StartExportSession")]
+        Task<StartExportSessionResponse> StartExportSessionAsync(StartExportSessionRequest request);
```
[the remark this answers is not in the document]
Comment (forsthug) [reply src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs:34]: true

File: src/fiskaltrust.ifPOS/v1/it/Models/FiscalReceiptRequest.cs (hunk 2 above)
Comment (forsthug) [reply src/fiskaltrust.ifPOS/v1/it/Models/FiscalReceiptRequest.cs:55]: its the payment are you sure? when divided payment /cash & card it would be here

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**